messages/protobuf/request_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019 NEC Laboratories Europe GmbH.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protobuf
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger-labs/minbft/messages"
+)
+
+func TestRequest(t *testing.T) {
```

Comment:
I guess that some test code using `isRequest()` to check the type elegantly will be added later?

Reply:
I think there is no need to test `isRequest()` because `impl.NewRequest()` simply cannot return anything not implementing `messages.Request` interface, which requires `isRequest()` method. It is checked by the compiler. Moreover, `isRequest()` is unexported in `messages` package, so it cannot be invoked here.

Reply:
OK, so this method is only used to make some type to fulfill the interface. How useful it is will be 
clearer with view-change patch.